doc/user/content/ingest-data/mongodb/_index.md

```diff
@@ -0,0 +1,334 @@
+---
+title: "MongoDB"
+description: "Connecting Materialize to a MongoDB database for Change Data Capture (CDC)."
+disable_list: true
+mermaid: true
+menu:
+  main:
+    parent: 'ingest-data'
+    identifier: 'mongodb'
+    weight: 17
+---
+
+Using Change Data Capture (CDC), you can track and propagate changes from
+MongoDB to downstream consumers. This guide outlines how to ingest data from a
+MongoDB replica set (`rs0`) into Materialize; specifically from the collections
+`items` and `orders` in the `test` database.
+
+## High-level architecture
+
+```mermaid
+flowchart LR
+    subgraph "Source"
+        Mongo[("MongoDB (Replica Set/Sharded Cluster)")]
+    end
+
+    subgraph "Streaming Ingestion: Debezium + Kafka"
+        Debezium["Debezium Connector"]
+        Kafka{{"Kafka Broker"}}
+        Schema["Schema Registry"]
+    end
+
+    subgraph "Materialize"
+        MZ[("Materialize")]
+    end
+
+    Mongo -- "Change stream oplog data" --> Debezium
+    Debezium -- "Writes Avro Events" --> Kafka
+    Schema -. "Validates Schemas" .- Kafka
+    Kafka -- "Consumes Topic" --> MZ
+    Schema -. "Decodes Data" .- MZ
+```
+
+The architecture consists of the following components:
+
+- **Source Component: MongoDB**
+  - **MongoDB**: MongoDB must be deployed as a **replica set** or **a sharded
+  cluster** to generate an oplog (operations log), which records all data
+  modifications. To deploy as a replica set or a sharded cluster, refer to the
+  official MongoDB documentation on [replica
+  sets](https://www.mongodb.com/docs/manual/replication/) and [sharded
+  clusters](https://www.mongodb.com/docs/manual/sharding/).
+
+- **Streaming Ingestion Component: Debezium + Kafka (with Schema Registry)**
+  - **Debezium:** A connector that [obtains the oplog data via MongoDB change
+    streams](https://debezium.io/documentation//reference/stable/connectors/mongodb.html#change-streams).
+  - **Kafka & Schema Registry:** The streaming transport layer. Debezium pushes
+    the events to a Kafka topic, while the Schema Registry ensures the data
+    structure (schema) is consistent and readable.
+
+- **Materialize:** Materialize uses connects to Kafka to ingests data from
+  MongoDB.
+
+## Prerequisites
+
+- **MongoDB:** Version 6.0 or later. Must be deployed as a [**replica
+  set**](https://www.mongodb.com/docs/manual/replication/) or a [**sharded
+  cluster**](https://www.mongodb.com/docs/manual/sharding/).
+
+- **Kafka Cluster:** A running Kafka broker and Schema Registry (e.g.,
+  Confluent Platform or Redpanda).
+
+- **Debezium Connect:** Version 2.5.4+ or later. A Kafka Connect cluster with
```

Comment:
Oh ... was able to test with v 3.1.2 ... but figured I'd leave it with the v2.5.4